yalla-coop/website-redesign
# Create Design System

Issue: #19 

## What did I do?
- Created initial folders and files of the design system
- Added **colors**, **font sizes**, **letter spacings**, and the **Karu font**
- Wrote some of the typography elements like **headings**, **link**, and **paragraph**

## How to test?
Run the app and you will see some of the elements that I've used in the first page, you can also run `yarn storybook` and you will be able to see the typography elements under the design system folder

Closes #19
